src/styles/index.css

```diff
@@ -146,8 +147,8 @@
     --p-color-divider: #51525C;
 
     --p-color-active: #1F647A;
-    --p-color-focus: #175CD3;
-    --p-color-focus-halo: rgba(23, 92, 211, 0.4);
+    --p-color-focus-ring: #276EE7;
+    --p-color-focus-ring-offset: var(--p-color-bg-0);
```

Comment:
hmm this is interesting. I'd expect this to be the bg of the layer its on. Or transparent. Can this be transparent?

Reply:
I'll try this again, but I believe transparent wasn't working.. doesn't make sense that it wouldn't though..

Reply:
`bg-0` is probably fine if transparent doesn't work but it stuck out to me as unexpected so thought I'd ask.

Reply:
Yeah so it's a nested shadow thing, the offset is another layered shadow on top to separate the outer shadow, not like a gradient thing like I was trying to imagine.

Reply:
Ha yeah that makes sense. This is fine the.